addons/hide-delete-button/addon.json

```
@@ -2,6 +2,12 @@
   "name": "Hide delete button",
   "description": "Hides delete button (trash can icon) from sprites, costumes and sounds. They can still be deleted using the right click context menu.",
   "tags": ["editor", "featured"],
+  "info": [
+    {
+      "text": "Bonus tip: Accidentally deleted a sprite, costume, or sound? You can bring it back with Edit \u2192 Restore <item> in the menu bar.",
```

```suggestion
      "text": "If you have accidentally deleted a sprite, costume, or sound, you can undo by selecting Edit \u2192 Restore <item> in the menu bar.",
```

---

Thanks! Here's my take, combining your suggestion:
```suggestion
      "text": "Bonus tip: If you accidentally delete a sprite, costume, or sound, you can undo the deletion by clicking Edit in the menu bar then Restore <item>.",
```
Do you think it should it say "bonus tip"? I thought that clarified that this is not relating to addon functionality.

---

Note to self: Update this in both places.

---

> Do you think it should it say "bonus tip"? I thought that clarified that this is not relating to addon functionality.

Maybe just "tip"?

---

Protip ™️